build/config.props

```diff
@@ -31,7 +31,7 @@
     <VsTargetChannelForTests>int.$(VsTargetBranch)</VsTargetChannelForTests>
 
     <!-- NuGet SDK VS package Semantic Version -->
-    <NuGetSdkVsSemanticVersion>$(VsTargetMajorVersion).$(MinorNuGetVersion).$(PatchNuGetVersion)</NuGetSdkVsSemanticVersion>
+    <NuGetSdkVsSemanticVersion>$([MSBuild]::Add(11, $(MajorNuGetVersion))).$(MinorNuGetVersion).$(PatchNuGetVersion)</NuGetSdkVsSemanticVersion>
```

Comment:
Can you explain this change?  The "before" code looks "better" to me in that it reuses instead of duplicating.